dcpquery/etl/__init__.py

```diff
@@ -110,7 +110,44 @@ def load_bundle(self, bundle, extractor=None, transformer=None):
         config.db_session.commit()
 
 
-def create_view_tables(extractor):
+def update_process_join_table():
+    config.db_session.execute(
+        """
+        Insert into process_join_table (child_process_uuid, parent_process_uuid)
+select * from (
+              with
+                  input_files_table as (
+                                        select
+                                               process_uuid as child_process,
+                                               file_uuid    as input_file_uuid
+                                        from process_file_join_table
+                                        where process_file_connection_type = 'INPUT_ENTITY'
+                                      ),
+
+                  output_files_table as (
+                                          SELECT
```

Comment:
Please fix capitalization to be consistent within the SQL statement.

Reply:
👍